docs/template.html

```diff
@@ -41,9 +41,15 @@
       }
     </script>
 
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.2/css/bootstrap.min.css" rel="stylesheet">
-    <link href="https://cdn.jsdelivr.net/gh/dokku/dokku@v0.27.4/docs/assets/style.css" rel="stylesheet">
-    <!-- <link href="/dokku/docs/assets/style.css" rel="stylesheet"> -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+
+    <!-- Font (Lato) -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    
+    <!-- <link href="https://cdn.jsdelivr.net/gh/dokku/dokku@v0.27.4/docs/assets/style.css" rel="stylesheet"> -->
+    <link href="./assets/style.css" rel="stylesheet">
```

Comment:
This needs to be versioned on merge, just fyi.

Reply:
Sounds good. I just wasn't sure what to use currently as the latest css isn't on GitHub. I believe I can safely uncomment the line and the CSS would become available once the PR is merged. Right?

Reply:
Once we merge it, I'll need to release to get it out. The workflow here is rough for development :(

Reply:
Once we merge it, I'll need to release to get it out. The workflow here is rough for development :(